components/OAuth2.0/OauthHttpLocalServer.c

```diff
@@ -146,15 +145,15 @@ httpd_handle_t Spotify_StartWebServer()
  * @brief This function stops the web HttpdServerHandler for handling HTTPS requests.
  * @return Returns the HTTP HttpdServerHandler handle if it is started successfully, or NULL otherwise.
  */
-esp_err_t Spotify_StopSpotifyWebServer(httpd_handle_t HttpdServerHandler)
-{
-    return httpd_stop(HttpdServerHandler);
-}
+// esp_err_t StopWebServer(httpd_handle_t HttpdServerHandler)
+// {
+//     return httpd_stop(HttpdServerHandler);
+// }
 
 /**
  * @brief This function starts the mDNS service.
  */
-bool Spotify_StartMDNSService()
+static bool Oauth_StartMDNSService()
 {
```

Comment:
mdns `hostname` should be an input of this function

Reply:
done.